src/main/native/include/sysid/analysis/FeedbackControllerPreset.h

```diff
@@ -17,6 +17,8 @@ struct FeedbackControllerPreset {
   /** The conversion factor between volts and the final controller output. */
   double outputConversionFactor;
 
+  double outputVelocityTimeFactor;
```

Comment:
Can you add a comment above this explaining what it does? Maybe even clarify that the double above this one doesn't account for time.

Reply:
Does:
`The conversion factor for using controller output for velocity gains. This is necessary as some controllers do velocity controls with different time units.`

clearly explain it?

Reply:
Yeah that sounds good.